SDKPackageManager.ps1

```diff
@@ -0,0 +1,31 @@
+param([Parameter(Mandatory=$true)][string]$arch, [Parameter(Mandatory=$true)][string]$config, [Parameter(Mandatory=$true)][string]$task)
+
+$currentLocation = Get-Location
+$packagePath = "$currentLocation\BuildOutput\$config\$arch\"
+
+if(!$task.CompareTo("i"))
```

Comment:
"i" and "u"? Not very powershell-esque

`--Install` and `--Uninstall` are more appropos.

```
Param(
    [Switch]$Install=$false,
    [Switch]$Unnstall=$false
)

if (($Install -eq $false) -And $($Uninstall -eq $false)) 
{
    Write-Output "Invalid arguments, aborting..."
    Exit 1
}
```

You could make 1 parameter as an enum e.g. 

```
Param(
    [ValidateSet("Install", "Uninstall")]
    [String]$Task
)
```

though this "-foo bar" feels a bit more irksome than the style above "-bar"

See `tools\TerminalVelocity\Generator-TerminalVelocityFeatures.ps1` for more ValidateSet examples